compiler/passes/InitNormalize.cpp

```diff
@@ -1851,10 +1851,14 @@ void InitNormalize::fieldInitFromField(Expr* insertBefore) {
   DefExpr* field = mCurrField;
 
   if        (field->exprType == NULL && field->init == NULL) {
-    USR_FATAL_CONT(insertBefore,
-                   "can't omit initialization of field \"%s\", "
-                   "no type or default value provided",
-                   field->sym->name);
+    if (toEnumType(field->sym->type) != NULL) {
+      USR_FATAL(field, "can't handle enums local to a class/record/union yet");
```

Comment:
It didn't break anything in testing...  My assumption is that a field of enum type would not have gotten past the previous conditional:

```chapel
  if (field->exprType == NULL && field->init == NULL) {
```

Thanks for the `isEnumType` tip.  I didn't realize we had that in the compiler (or forgot)...

Reply:
It would be fine with me if the PR was merged as-is. I still don't think it's quite the right place to have the error, but I'd rather not have the compiler segfault :)

If you do stick with this branch, could you just add a TODO comment?